CHANGELOG.md

```diff
@@ -58,7 +60,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 [#839]: https://github.com/knurling-rs/defmt/pull/839
 [#838]: https://github.com/knurling-rs/defmt/pull/838
 
-## defmt-macros v0.3.9
+### defmt-macros v0.3.9
```

Comment:
Why did this change a level? It feels like it was a release of some crates in and of itself, even if the main defmt crate wasn't bumped.

Reply:
I thought it makes sense to highlight the defmt releases more than the related releases. But we can change it back if you don't like it.

Reply:
to me, it implied that the `###` releases belonged to their parent `##` release, which didn't seem right

Reply:
we could maybe use emojis to mark the defmt releases, which I agree are a bigger deal. No idea which one though.

Reply:
I changed it back. The defmt releases are highlighted by the link in the heading.